### ISSUE TITLE
74 Host - Network Redundancy lost - obsolete command

### ISSUE THREAD

**Issue by hbstreve** [alanrenouf/vCheck-vSphere#745]
Running the main vCheck, watching the output, I came across this warning

![image](https://user-images.githubusercontent.com/64648191/177388894-6d2c0a75-8495-4290-a97f-cc43b05825b7.png)

Appears it needs to be updated with the newer Get-VDSwitch command

**Comment by rebelinux** [comment]
Fix it by adding -Standard to the to the Get-VirtualSwitch cmdlet

```
PS C:\Users\jocolon\vCheck-vSphere> .\vCheck.ps1
WARNING:
Begin Plugin Processing
[18:05:54] ..start calculating Connection settings for vCenter by Alan Renouf v1.20 [1 of 3]
[18:06:02] ..finished calculating Connection settings for vCenter by Alan Renouf v1.20 [1 of 3]
[18:06:02] ..start calculating General Information by Alan Renouf, Frederic Martin v1.3 [2 of 3]
[18:06:23] ..finished calculating General Information by Alan Renouf, Frederic Martin v1.3 [2 of 3]
[18:06:23] ..start calculating Network redundancy lost by Olivier TABUT v1.4 [3 of 3]
[18:06:24] ..finished calculating Network redundancy lost by Olivier TABUT v1.4 [3 of 3]
[18:06:24] ..Displaying HTML results
PS C:\Users\jocolon\vCheck-vSphere>
```

Tested in my homelab:

![NetworkHostRedundancy](https://user-images.githubusercontent.com/1002783/198903921-28d31bd2-60ef-4dfc-adc0-217768f7f200.png)
